tests/cross-language/go/decrypt_test.go

```diff
@@ -0,0 +1,88 @@
+package cltf
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/godaddy/asherah/go/appencryption"
+	"github.com/godaddy/asherah/go/appencryption/pkg/crypto/aead"
+	"github.com/godaddy/asherah/go/appencryption/pkg/kms"
+	"github.com/godaddy/asherah/go/appencryption/pkg/persistence"
+	"github.com/pkg/errors"
+)
+
+var (
+	decryptedPayload string
+	encryptedPayload string
+)
+
+func ihaveencryptedDatafrom(filename string) error {
+	data, err := ioutil.ReadFile(fmt.Sprintf("%s%s", FileDirectory, filename))
+	if err != nil {
+		panic(err)
+	}
+
+	encryptedPayload = string(data)
+
+	return nil
+}
+
+func idecrypttheencryptedData() error {
+	crypto := aead.NewAES256GCM()
+	manager, _ := kms.NewStatic(KeyManagementStaticMasterKey, crypto)
+	metastore := persistence.NewSQLMetastore(connection)
+	policy := appencryption.NewCryptoPolicy()
+	config := &appencryption.Config{
+		Service: DefaultServiceID,
+		Product: DefaultProductID,
+		Policy:  policy,
+	}
+
+	factory := appencryption.NewSessionFactory(config, metastore, manager, crypto)
+	defer factory.Close()
+
+	sess, err := factory.GetSession(DefaultPartitionID)
+	if err != nil {
+		panic(err)
+	}
+	defer sess.Close()
+
+	var dataRow appencryption.DataRowRecord
+
+	dataRowBytes, err := base64.StdEncoding.DecodeString(encryptedPayload)
+	if err != nil {
+		panic(err)
+	}
+
+	err2 := json.Unmarshal(dataRowBytes, &dataRow)
```

Comment:
can just use `err`

Reply:
Shadows the previous err, if used.

Reply:
the previous error has already been handled. can clarify, how it is being shadowed.

Reply:
It's definitely safe to reuse `err` here as their both usages are in the same scope so there's no shadowing. Just use the regular assignment operator (`err = ...`) instead of the short declaration operator (`err := ...`) and you'll be good.